File: app/src/main/java/workshop/akbolatss/xchangesrates/base/ErrorView.java
package workshop.akbolatss.xchangesrates.base;

/**
 * Author: Akbolat Sadvakassov
 * Date: 27.12.2017
 */

public interface ErrorView {

    public void onUnknownError(String errorMessage);

    public void onTimeout();

    public void onNetworkError();
}

File: app/src/main/java/workshop/akbolatss/xchangesrates/base/ErrorView.kt
package workshop.akbolatss.xchangesrates.base

/**
 * Author: Akbolat Sadvakassov
 * Date: 27.12.2017
 */

interface ErrorView {

    fun onUnknownError(errorMessage: String)

    fun onTimeout()

    fun onNetworkError()
}
